comet/scrapers/animetosho.py

```
@@ -0,0 +1,110 @@
+        return torrents
+
+    async def scrape_page(self, query, offset, limit):
+        try:
+            async with self.session.get(
+                f"https://feed.animetosho.org/api?t=search&q={query}&offset={offset}&limit={limit}"
+            ) as response:
+                content = await response.text()
+                if not content.strip():
+                    return [], 0
+
+                root = ET.fromstring(content)
```

Ruff (0.14.10)

Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)